src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -296,6 +314,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       model.customResponseVisibleTime = customResponseVisible.time;
       model.customResponseVisibleDate = customResponseVisible.date;
     }
+    this.feedbackSessionModelBeforeEditing = feedbackSession;
```

Comment:
I don't think the line should be added here. `getSessionEditFormModel` should not have any side effect.